src/explore-education-statistics-frontend/src/modules/table-tool/components/FixedMultiHeaderDataTable.tsx

```diff
@@ -61,7 +61,12 @@ const FixedMultiHeaderDataTable = forwardRef<HTMLElement, Props>(
             ref={mainTableRef}
           />
         </div>
-        {footnotes && <Footnote content={footnotes} />}
+        {footnotes && (
+          <div>
```

Comment:
can we not use `<>` rather than `<div>` ?

Reply:
Changed it